core/domain/config_domain.py

```diff
@@ -334,10 +344,10 @@ def get_config_property(cls, name):
         Returns:
             instance. The instance of the specified configuration property.
         """
-        return cls._config_registry.get(name)
+        return cls._config_registry.get(name) # type: ignore[return-value]
```

Comment:
Why is this ignore needed?

Reply:
This function returns object of `configProperty` but if remove this ignore statement, then mypy expect that function returns optional[configProperty]
which also generates the error:
 `core/domain/config_domain_test.py:100: error: Item "None" of "Optional[ConfigProperty]" has no attribute "schema"  [union-attr]`
but we know there is always a schema attribute.

Reply:
I think you can remove this ignore and then fix this in the tests.

Reply:
casting this into `configProperty` fixed this issue!!
done!